include/secp256k1.h

```diff
@@ -526,6 +526,37 @@ SECP256K1_API extern const secp256k1_nonce_function secp256k1_nonce_function_rfc
 /** A default safe nonce generation function (currently equal to secp256k1_nonce_function_rfc6979). */
 SECP256K1_API extern const secp256k1_nonce_function secp256k1_nonce_function_default;
 
+/** Compute commitment on the client as part of the ECDSA Anti Nonce Sidechannel Protocol.
+ *
+ * ECDSA Anti Nonce Sidechannel Protocol:
+ * 1. The host draws randomness `k2`, commits to it with sha256 and sends the commitment to the client.
+ * 2. The client commits to it's original nonce `k1` using the host commitment by calling
+ *    `secp256k1_ecdsa_anti_nonce_sidechan_client_commit`. The client sends the resulting commitment
+ *   `R1` to the host.
+ * 3. The host replies with `k2` generated in step 1.
+ * 4. The client checks that the host commitment from step 1 commits to `k2` from step 3 and signs
+ *    with `secp256k1_ecdsa_sign_nonce_tweak_add`, using the `k2` as the tweak and the host
```

Comment:
In general I'm trying to avoid the name `tweak` because it carries little information.

Reply:
fixed